public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -353,26 +356,39 @@ export class ElasticDatasource extends DataSourceApi<ElasticsearchQuery, Elastic
   }
 
   interpolateVariablesInQueries(queries: ElasticsearchQuery[], scopedVars: ScopedVars): ElasticsearchQuery[] {
```

Comment:
I didn't add tests for this as it is already tested and *should* have a good coverage